src/tests/specs/ink/Misc.spec.ts

```diff
@@ -196,8 +196,7 @@ C C C
   // TestReturnTextWarning
   it("tests return text warning", () => {
     compileStoryWithoutRuntime("return_text_warning");
-
-    expect(context.warningMessages.length).toBe(1);
+    expect(context.warningMessages.length).toBe(2);
```

Comment:
Well, seems that upstream doesn't check the number of warnings, so I guess `toBeGreaterThanOrEqual(1)` would work just as well!